extensions/amp-anim/0.1/amp-anim.js

```diff
@@ -81,9 +81,9 @@ export class AmpAnim extends AMP.BaseElement {
 
   /** @override */
   viewportCallback(inViewport) {
-    if (!inViewport || !this.loadPromise_) {
+    if (!inViewport) {
```

Comment:
I don't think this condition is necessary now.

This current code does: 

- `inViewport = true`
  - if `loadPromise`, set image to visible
  - else, keep image hidden
- `inViewport = false`
  - if `loadPromise`, set image to hidden (this one's not explicit)
  - else, set image to hidden

That's functionally the same as:

- if `loadPromise`
  - if `inViewport = true`, set image to visible
  - if `inViewport = false`, set image to hidden